db/migrations/0002_guild_alter_race_name_skill_player.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 4.0.2 on 2023-12-24 05:41
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Guild',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+                ('description', models.TextField(blank=True, null=True)),
+            ],
+        ),
+        migrations.AlterField(
+            model_name='race',
+            name='name',
+            field=models.CharField(choices=[('elf', 'Elf'), ('dwarf', 'Dwarf'), ('human', 'Human'), ('ork', 'Ork')], max_length=255, unique=True),
+        ),
+        migrations.CreateModel(
+            name='Skill',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+                ('bonus', models.CharField(max_length=255, unique=True)),
+                ('race', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.race')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Player',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('nickname', models.CharField(max_length=255, unique=True)),
+                ('email', models.EmailField(max_length=255, unique=True)),
+                ('bio', models.CharField(max_length=255)),
+                ('guild', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='db.guild')),
+                ('race', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.race')),
+            ],
+        ),
+    ]
```

Comment:
It is a good practice to include one migration per app with all required changes to the models in one pull request to not overload the `migrations` folder in a Django project. Include only one migration in this pull request.